pkg/cmd/tag-merged-pr/main.go

```diff
@@ -0,0 +1,240 @@
+// Copyright 2018 The Cockroach Authors.
```

Comment:
Since these are all new files, this date should be 2021 for all the new files containing the notice.

```
// Copyright 2021 The Cockroach Authors.
```